binancial/utils/__init__.py

```
@@ -1 +1,3 @@
-from .add_klines_features import add_klines_features
\ No newline at end of file
+from .add_klines_features import add_klines_features
```

Ruff (0.8.2)

`.add_klines_features.add_klines_features` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)